TicTacToe.py

```diff
@@ -92,7 +91,8 @@ def updateBoard(self, row, col, player_icon):
 			self.board[row][col] = player_icon
 			self.checkBoard()
 		else:
-			err = f"Tried to update the board with '{player_icon}' but the only choices are '{self.BLANK_POS_ICON}', '{self.PLAYER_0_ICON}', and '{self.PLAYER_1_ICON}'."
+			err = f"Tried to update the board with '{player_icon}' but the only choices are"
+			err += "'{self.BLANK_POS_ICON}', '{self.PLAYER_0_ICON}', and '{self.PLAYER_1_ICON}'."
```

Comment:
Well... that's one way to do it 😂 I think we may change this later on, there has to be a more "Pythonic" way to do that, but it works for now

Reply:
I could change it to be multi-line with a \

Reply:
```suggestion
			err = (f"Tried to update the board with '{player_icon}' but the only choices are "
       f"'{self.BLANK_POS_ICON}', '{self.PLAYER_0_ICON}', and '{self.PLAYER_1_ICON}'.")

```

This is the recommended way of breaking up long lines by PEP8.